packages/blockchain/src/index.ts

```diff
@@ -128,14 +136,30 @@ export default class Blockchain implements BlockchainInterface {
       })
     }
 
-    this._validatePow = opts.validatePow !== undefined ? opts.validatePow : true
+    this._validateConsensus = opts.validateConsensus !== undefined ? opts.validateConsensus : true
     this._validateBlocks = opts.validateBlocks !== undefined ? opts.validateBlocks : true
 
     this.db = opts.db ? opts.db : level()
     this.dbManager = new DBManager(this.db, this._common)
 
-    if (this._validatePow) {
-      this.ethash = new Ethash(this.db)
+    if (this._validateConsensus) {
+      if (this._common.consensusType() !== 'pow') {
```

Comment:
Yes, that's a good point. We have an open issue for that at #702, this has (somewhat) high demand and this might still be a candidate to integrate along the dev releases. I wouldn't want to side-introduce along the work here, is it ok if we do this as some separate work along the issue from above? I will directly go over, update the issue and place some open questions I still have on this.

Reply:
See https://github.com/ethereumjs/ethereumjs-vm/issues/702#issuecomment-722296424